kwalitee/cli.py

```diff
@@ -81,6 +84,26 @@ def generate_json(ctx, output=None):
     else:
         print(json_str)
 
+
+@cli.command('releases', help='''
+
+    Generate JSON for ocrd_all releases
+
+''')
+@click.option('-o', '--output', help="Output file. Omit to print to STDOUT")
```

Comment:
```suggestion
@click.option('-o', '--output', help="Output file. '-' to print to STDOUT", type=click.File('w', encoding='utf-8'))
```

Reply:
Agree, but we should add this to the other CLI options as well.